src/hb-set.hh

```diff
@@ -506,16 +506,34 @@ struct hb_set_t
 
   bool is_subset (const hb_set_t *larger_set) const
```

Comment:
Can you update the formatting of the new code to match the style in this file. Notably:
- 2 space indent
- if( -> if (
- function_call( -> function_call (
- { should be on their own line

Reply:
Done